locales/en.default.json

```diff
@@ -74,7 +74,8 @@
     "loading": "Loading...",
     "total_reviews": "total reviews",
     "star_reviews_info": "{{ rating_value }} out of {{ rating_max }} stars",
-    "collapsible_content_title": "Collapsible content"
+    "collapsible_content_title": "Collapsible content",
+    "open_in_new_window": "Opens in a new window."
```

Comment:
We already have this string under `link_messages.new_window` just above (line 71).